### PR TITLE
Add healthcheck view test and fix contact us log typo

### PR DESCRIPTION
**Pull request by NolanVH**
### Description of Changes
This PR adds a health check endpoint test and a typo fix for a contact-us email log message has been added.

### Checklist
- ~README / documentation has been updated if needed~
- ~PR title / commit messages contain Jira ticket number if applicable~
- ~Tests have been updated / created if applicable~
- ~Admin Panel has been updated if models have been added or modified~
- ~Migrations have been updated and committed if applicable~
- ~Team Postman workspace has been updated if applicable~

### Reviewers Should Do The Following:
- [x] Review the code changes here
- [ ] Pull the branch and test locally

### Additional Notes
N/A

### CODE REVIEW

**Review by bronson-c — APPROVED** [review]
LGTM, great change!

**Review by sahil97 — APPROVED** [review]
LGTM !

File: firstvoices/healthcheck/views.py [hunk 1]
```diff
@@ -1,11 +1,22 @@
+import subprocess
+
 from rest_framework import permissions
 from rest_framework.response import Response
 from rest_framework.views import APIView
 
 
+def get_git_revision_hash() -> str:
+    return subprocess.check_output(["git", "rev-parse", "HEAD"]).decode("ascii").strip()
```
[the remark this answers is not in the document]
Comment (NolanVH) [reply firstvoices/healthcheck/views.py:9]: I just double-checked and it looks like git is not installed but the `.git` directory with the information is there. It should work if we install git. I will send a message to Rob. Good catch, thanks!

[the remark this answers is not in the document]
Comment (NolanVH) [reply firstvoices/healthcheck/views.py:9]: After a chat with Rob the plan is to set the commit hash in an environment variable during the build and then pull from the variable in this view. I will set the PR to a draft until that change has been made.

**Review by sarahfirstvoices — APPROVED** [review]

**Pull request merged**